spec/lib/rmt/cli/products_spec.rb

```diff
@@ -380,6 +380,7 @@
       output += "Disabling #{product.friendly_name}:\n"
       output += "  #{product.friendly_name}:\n"
       repos.pluck(:name).sort.each { |repo| output += "    Disabled repository #{repo}.\n" } unless repos.empty?
+      output += "\n\e[1mTo clean up downloaded files, please run 'rmt-cli repos clean'\e[22m\n"
```

Comment:
We shouldn't use "please", as per our [docu guidelines](https://documentation.suse.com/style/current/single-html/docu_styleguide/index.html#sec-vocabulary):

```suggestion
      output += "\n\e[1mTo clean up downloaded files, run 'rmt-cli repos clean'\e[22m\n"
```